path_config.go

```diff
@@ -64,12 +68,27 @@ func pathConfig(b *azureSecretBackend) *framework.Path {
 				Type:        framework.TypeBool,
 				Description: "Enable usage of the Microsoft Graph API over the deprecated Azure AD Graph API.",
 			},
+			"default_expiration": &framework.FieldSchema{
+				Type: framework.TypeString,
```

Comment:
Can this be `framework.TypeDurationSecond` and let the framework handle the parsing?